tools/replay/desktop_main.cpp

```diff
@@ -52,7 +57,7 @@
 
 extern "C"
 {
-    __declspec(dllexport) extern const UINT D3D12SDKVersion = 606;
+    __declspec(dllexport) extern const UINT D3D12SDKVersion = 602;
```

Comment:
Should this still be `606`? Has the Agility SDK version changed?

Reply:
Yeah, this should stay as `606`